sim/midas/src/main/cc/simif.h

```diff
@@ -155,26 +172,72 @@ class simif_t {
 
   // End host-platform interface.
 
-  // LOADMEM functions
-  void read_mem(size_t addr, mpz_t &value);
-  void write_mem(size_t addr, mpz_t &value);
-  void write_mem_chunk(size_t addr, mpz_t &value, size_t bytes);
-  void zero_out_dram();
+  /// Returns the seed of the random-number generator.
```

Comment:
I'd prefer if we'd rally about javadoc/scaladoc variants of doxygen comments if possible.